test/utils/initMatchers.js

```diff
@@ -35,7 +35,7 @@ chai.use((chaiAPI, utils) => {
       currentNode !== document.documentElement &&
       ariaHidden === false
     ) {
-      ariaHidden = element.getAttribute('aria-hidden') === 'true';
+      ariaHidden = currentNode.getAttribute('aria-hidden') === 'true';
```

Comment:
:man_facepalming: 

Good news is that this functionality will be moved into testing-library. We'll probably expose some `isAccessible` utility which we can use.